scaleway/vcr_config.py

```diff
@@ -0,0 +1,127 @@
+import os
+import json
+from pathlib import Path
+from urllib.parse import urlparse, parse_qs, urlencode, urlunparse
+
+import vcr
+import inspect
+
+PYTHON_UPDATE_CASSETTE = os.getenv("PYTHON_UPDATE_CASSETTE", "false").lower() in (
+    "1",
+    "true",
+    "yes",
+)
+
+CI = os.getenv("CI", "false").lower() in (
```

Comment:
I'm a little skeptical here because the nightly also are running in a CI AND it would be passed to false then. It would be helpful to find a different name.